### PR TITLE
#1: [trello] sync issues with labels "customers" with Trello card (closes #1)

### LINKED ISSUE #1: [trello] sync issues with labels "customers" with Trello card
## requirements
- any issue with label "customers" should have an associated Trello card
- only issues with labels "customers" should have an associated Trello card

## specs
- listen to GitHub webhooks:
  - check if issue has label "customers"
    - yes -> add/update card
    - no -> if it exists, delete card

### PR DESCRIPTION
**Pull request by FrancescoCioria**
Issue #1

### CODE REVIEW

File: src/processors/trello.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+async function getBacklogId(repoName) {
+  const { trello, boardId } = trelloConfigForRepo(repoName);
+  return trello.getListsOnBoard(boardId)
+    .then(lists => Promise.resolve(find(lists, { name: 'backlog' }).id));
+}
+
+async function getDoneId(repoName) {
+  const { trello, boardId } = trelloConfigForRepo(repoName);
+  return trello.getListsOnBoard(boardId)
+    .then(lists => Promise.resolve(find(lists, { name: 'done' }).id));
```
Comment (gabro) [inline src/processors/trello.js]: No need for  Promise.resolve inside a then

File: src/processors/trello.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  }
+}
+
+async function archiveOrUnarchiveCard(repoName, issue) {
+  const card = await getCardIfItExists(repoName, issue);
+  const hasCustomerLabel = hasLabel(issue, labels.customers);
+  const { trello } = trelloConfigForRepo(repoName);
+
+  if (card && card.closed && hasCustomerLabel) {
+    // TODO: never called as getCardIfItExists doesn't work for archived ones
```
Comment (FrancescoCioria) [inline src/processors/trello.js]: I'll fix this in https://github.com/buildo/prism/issues/3 (pretty big and not relevant for this iteration)

File: src/index.js [hunk 3]
```diff
@@ -1,6 +1,8 @@
 import express from 'express';
 import bodyParser from 'body-parser';
 import Rx from 'rx';
+import 'babel-core/register';
```
Comment (gabro) [inline src/index.js]: why babel-core register?

Comment (FrancescoCioria) [inline src/index.js]: It's required to use async / await

Comment (gabro) [inline src/index.js]: Oh, I see. Thanks

**Pull request merged**